config/app.yml

```diff
@@ -22,6 +22,8 @@ all:
 
   # Google Analitycs
   # google_analytics_api_key: ...
+  # google_analytics_institutions_dimension_index: 1
```

Comment:
Perhaps a comment indicating what this setting is for:  e.g. a value of zero disables the institution dimension.